src/test/java/TddWorldTest.java

```diff
@@ -88,19 +87,14 @@ public void testOdd() {
 	@Test
```

Comment:
We can also make use of DisplayName("Describe that I am doing this") to make it more readable for devs

Reply:
Done!